src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,21 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        StringBuilder builder = new StringBuilder("");
+        int remainder = 0;
+        while (value >= 1) {
```

Comment:
let's change the condition to `value >= 0`, and then you can remove `if` statement